backend/src/test/java/com/github/mysterix5/vover/cloud_storage/CloudRepositoryTest.java

```diff
@@ -19,7 +19,7 @@ void find() throws IOException {
         String url = "https://cloud.com/";
         CloudRepository cloudRepository = new CloudRepository(mockedSardine, url);
 
-        var obFile = new File("src/test/resources/cloud_storage/ob.mp3");
+        var obFile = new File("src/test/resources/cloud_storage/eins.mp3");
         var obStream = new FileInputStream(obFile);
 
         Mockito.when(mockedSardine.get(url + "ob.mp3")).thenReturn(obStream);
```

Comment:
Interesting, what is mockedSardine ?

Reply:
Sardine is a library to send webdav requests very easily. I need it to access my cloud storage via webdav